discovery-dns/src/main/resources/reference.conf

```diff
@@ -2,14 +2,19 @@
 # Akka Service Discovery DNS Config                  #
 ######################################################
 
-akka.discovery {
+akka {
+  discovery {
 
-  # Set the following in your application.conf if you want to use this discovery mechanism:
-  # impl = akka-dns
+    # Set the following in your application.conf if you want to use this discovery mechanism:
+    # impl = akka-dns
 
-  # configured the akka-dns provider
-  akka-dns {
-    # currently no special configuration is present for this module
-    class = akka.discovery.dns.DnsSimpleServiceDiscovery
+    # configured the akka-dns provider
+    akka-dns {
+      # currently no special configuration is present for this module
+      class = akka.discovery.dns.DnsSimpleServiceDiscovery
+
+      # possible values: ip srv. Ip will do a A/AAAA DNS query. Srv will do an SRV DNS query.
+      lookup-type = srv
```

Comment:
This is confusing hm. I was under the impression one per each query knows if it's srv or normal, since srv records are supposed to start with an `_`

https://en.wikipedia.org/wiki/SRV_record

Reply:
Hmm, I'm dumb -- since this is auto discovery and name can be taken from system name... this will not have _...
However, I think it would be best if we issue both request perhaps -- try srv and if none found then issue an A request -- so people don't have to configure this at all.

WDYT?

Reply:
I think this relates to #224. If you pass a port name/protocol, do SRV. Otherwise, A record.

Reply:
Yes i'll rebase this on top of 224 and do that